python/ray/tests/test_stress_sharded.py

```diff
@@ -42,7 +42,7 @@ def g(n):
             x = f.remote(x)
         return x
 
-    ray.get([g(1000) for _ in range(100)])
+    ray.get([g(100) for _ in range(100)])
```

Comment:
This is the slowest of this suite. Stress sharded takes >10 min to run, try to make it a bit faster.